src/internal/operators/isEmpty.ts

```diff
@@ -3,6 +3,60 @@ import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
 import { OperatorFunction } from '../types';
 
+/**
+ * Emits true if the input observable emits any values, or emits false if the
+ * input observable completes without emitting any values.
+ *
+ * <span class="informal">Tells whether any values are emitted by an observable</span>
+ *
+ * ![](isEmpty.png)
+ *
+ * `isEmpty` transforms an Observable that emits values into an Observable that
+ * emits a single boolean value representing whether or not any values were
+ * emitted by the source Observable. As soon as the source Observable emits a
+ * value, `isEmpty` will emit a `true`.  If the source Observable completes having
+ * not emitted anything, `isEmpty` will emit a `false`.
+ *
+ * A similar effect could be achieved with {@link count}, but `isEmpty` can emit
+ * a `false` value sooner.
+ *
+ * ## Examples
+ *
+ * Emit `false` for a non-empty Observable
+ * ```javascript
+ * import { from } from 'rxjs';
+ * import { isEmpty, shareReplay } from 'rxjs/operators';
+ *
+ * const values = from(['a', 'b', 'c']).pipe(shareReplay());
+ * const result = values.pipe(isEmpty());
+ * values.subscribe(x => console.log(x));
+ * result.subscribe(x => console.log(x));
+ * // Results in:
+ * // a
+ * // false
+ * // b
+ * // c
```

Comment:
The result of this example is different (maybe this one comes from RxJS 4 examples?). It first prints letters `a`, `b` and `c` and then prints `false`.

Reply:
Fixed the example.